testsuite/integration/elytron/pom.xml

```diff
@@ -61,6 +61,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
```

Comment:
This dependency is needed for OCP tests to work. In WF 26 this dependency is coming here from somewhere but is not directly in POM I'm currently searching from where it is in WF 26.

Reply:
In WF 26 it is imported from:
[INFO] +- org.wildfly.core:wildfly-model-test:jar:18.1.2-8-PSI-SNAPSHOT:compile
[INFO] |  +- com.google.guava:failureaccess:jar:1.0.1:compile

The question is should It be added there or should it be added directly here